pandas/util/_doctools.py

```diff
@@ -41,7 +43,7 @@ def _get_cells(self, left, right, vertical) -> tuple[int, int]:
             hcells = sum([self._shape(df)[1] for df in left] + [self._shape(right)[1]])
         return hcells, vcells
 
-    def plot(self, left, right, labels=None, vertical: bool = True):
+    def plot(self, left, right, labels: Sequence[str] = (), vertical: bool = True):
```

Comment:
😱

Reply:
Technically, it could even be Iterable[str] (only used within zip). And yes a string is okay for this function :)

Changed the default value as there is no check whether labels is None. Would raise if labels is not provided.

Reply:
changed to `Iterable[str]` (still contains str and even scarier types)